Testing/Stereo-Inertial/README.md

```diff
@@ -0,0 +1,59 @@
+## Testing Stereo-Inertial SLAM using existing datasets
+Currently, this repo supports testing against the EuRoC dataset.
+
+### Installing the datasets
+To install the EuRoC dataset, visit https://projects.asl.ethz.ch/datasets/doku.php?id=kmavvisualinertialdatasets. There are multiple test sequences with varying difficulty to choose from.
+
+### Running tests on existing datasets
+`run_stereo_inertial_tests.sh` contains examples of how to run tests on existing datasets.
+
+To run tests on the EuroC dataset:
+```bash
+# Testing/Stereo-Inertial/
+../../bin/test_stereo_inertial_euroc <path_to_vocabulary> <path_to_euroc_camera_settings> <path_to_euroc_sequence_folder> <path_to_output_csvfile>
+```
```

Comment:
Please clarify that this is what the run_stereo will do and that this example is an example of what `run_stereo_inertial_tests.sh` will do.